pallets/pallet-did-lookup/Cargo.toml

```diff
@@ -21,6 +21,11 @@ sp-keystore = {branch = "polkadot-v0.9.19", git = "https://github.com/paritytech
 codec = {package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive"]}
 log = "0.4.16"
 scale-info = {version = "2.1.1", default-features = false, features = ["derive"]}
+sha3 = { version = "0.9", default-features = false }
```

Comment:
Can you sort dependencies in each section in alphabetical order?

Reply:
done